Chat/Defines.h

```diff
@@ -7,7 +7,7 @@
 #define BROADCAST_IP "255.255.255.255"
 //max buffer size without
 //possible data tearing
-#define BUFFER_SIZE 512
+#define BUFFER_SIZE 4092
```

Comment:
Why 4092?
I suppose, it is 1 kilobyte multiplied by four.
If so, we have 1024 * 4 = 4096
Strange...